src/main/kotlin/com/launchdarkly/intellij/action/ChangeFallthroughAction.kt

```diff
@@ -111,9 +112,9 @@ class ChangeFallthroughAction : AnAction {
     override fun update(e: AnActionEvent) {
         super.update(e)
         val project = e.project ?: return
-        if (project.service<FlagToolWindow>().getPanel().tree.lastSelectedPathComponent != null) {
+        if (project.service<FlagToolWindow>().getPanel().getFlagPanel().tree.lastSelectedPathComponent != null) {
             val selectedNode =
-                project.service<FlagToolWindow>().getPanel().tree.lastSelectedPathComponent.toString()
+                project.service<FlagToolWindow>().getPanel().getFlagPanel().tree.lastSelectedPathComponent.toString()
```

Comment:
As per my comments above, we can refactor these with the new getSelectedNode function.